website/docs/r/purview_account.html.markdown

```diff
@@ -73,6 +75,16 @@ A `identity` block exports the following:
 
 * `type` - The type of Managed Identity assigned to this Purview Account.
 
+---
+
+A `managed_resources` block exports the following:
+
+* `event_hub_namespace` - The ID of the managed event hub namespace.
+
+* `resource_group` - The ID of the managed resource group.
+
+* `storage_account` - The ID of the managed storage account.
```

Comment:
since these are all ID's, can we suffix these with `_id`:

```suggestion
* `event_hub_namespace_id` - The ID of the managed event hub namespace.

* `resource_group_id` - The ID of the managed resource group.

* `storage_account_id` - The ID of the managed storage account.
```